src/lib/y2storage/btrfs_subvolume.rb

```diff
@@ -77,6 +77,78 @@ def set_default_btrfs_subvolume
       to_storage_value.public_send(:default_btrfs_subvolume=)
     end
 
+    # Whether the filesystem can be shadowed
+    # @return [Boolean]
+    def can_be_shadowed?
```

Comment:
Not sure if the name is self-explanatory enough. Strictly speaking all subvolumes can be shadowed (although they probably shouldn't). Is not clear enough that this attribute is about automatic deletion.